lesson_17/tests/test_human.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from lesson_17.Human.human import Human
+
+
+@pytest.mark.parametrize("name,age,gender", [('Ann', 1, "female"), ('Bob', 99, "male")])
+@pytest.mark.positive
+def test_create_human(name, age, gender):
+    human = Human(name, age, gender)
```

Comment:
you should not create Human directly in the test. much better to implement such fixture